near-accounts/examples/function_call.rs

```diff
@@ -27,11 +28,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let args_json = json!({"message": "working1"});
 
+    let t1 = time::Instant::now();
     let result = account
         .function_call(&contract_id, method_name, args_json, gas, 0)
+        .await?
+        .transact()
         .await;
-
+    let t2 = time::Instant::now();
     println!("response: {:#?}", result);
+    println!("Time taken: {:?}",t2-t1);
```

Comment:
[suggestion] you should use **t1.elapsed**  to find out time taken by the function.